datadog_checks_base/datadog_checks/base/checks/base.py

```diff
@@ -333,10 +333,12 @@ def set_external_tags(self, external_tags):
         #     ('hostname2', {'src2_name': ['test2:t3']})
         # ]
         try:
-            for _, source_map in external_tags:
+            new_tags = []
```

Comment:
Didnt @masci factor this out?

Reply:
We're waiting for 6.12 to release to merge it IIUC.